cfme/utils/dockerbot/check_prs.py

```diff
@@ -15,13 +15,13 @@
 from cfme.utils.log import setup_logger
 from cfme.utils.trackerbot import api
 
-token = docker_conf['gh_token']
-owner = docker_conf['gh_owner']
-repo = docker_conf['gh_repo']
+token = docker_conf.get('gh_token')
```

Comment:
I made this change so these wouldn't raise KeyError on module import, which happens during RTD.

Alternatively I can include the docker.yaml.template conversion in docs/conf.py.